setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
That will be New Year wish for all of you guys to never have to do "shebang.bat" and related "magic" of that caliber! ;-)